NOTES-NONSTOP.md

```diff
@@ -0,0 +1,151 @@
+ * For 3.0 `--prefix=/usr/local-ssl3.0 --openssldir=/usr/local-ssl3.0/ssl` 
+ * For 3.0 PUT `--prefix=/usr/local-ssl3.0_put --openssldir=/usr/local-ssl3.0_put/ssl`
+
+ Use the `_RLD_LIB_PATH` environment variable in OSS to select the appropriate
+ directory containing `libcrypto.so` and `libssl.so`. In GUARDIAN, use the
+ `=_RLD_LIB_PATH` search define to locate the GUARDIAN subvolume where OpenSSL
+ is installed.
+
+Float Considerations
+--------------------
+
+ OpenSSL is built using IEEE float mode by default. If you need a different
+ IEEE mode, create a new configuration specifying `tfloat-x86-64` (for Tandem
+ float) or `nfloat-x86-64` (for Neutral Float).
+ 
+Memory Models
+-------------
+
+ The current OpenSSL default memory model uses the default platform address
+ model. If you need a different address model, you must specify the appropriate
+ c99 options for compile (`CFLAGS`) and linkers (`LDFLAGS`).
+
+Cross Compiling with NSDEE
+--------------------------
+
+ **Note:** None of these builds have been tested by the platform maintainer and are
+ supplied for historical value. Please submit a Pull Request to OpenSSL should
+ these need to be adjusted.
+ 
+ If you are attempting to build OpenSSL with NSDEE, you will need to specify
+ the following variables. The following set of compiler defines are required:
+ 
+    COMP_ROOT=$(cygpath -w /path/to/comp_root)   # must be path format for system (ie windows)
+    CC=/path/to/c99                              # must be executable by shell
+ 
+### Optional Build Variables 
+
+    DBGFLAG="--debug"
+    CIPHENABLES="enable-ssl3 enable-ssl3-method enable-weak-ssl-ciphers enable-rc4"
+
+### Internal Known On Platform Cross Compile Variables
+
+  The following definition is required if you are building on TNS/X for TNS/E
+  and have access to a TNS/E machine on your EXPAND network - with an example
+  node named \CS3:
+   
+    SYSTEMLIBS="-L/E/cs3/usr/local/lib"
+
+Version Procedure (VPROC) Considerations
+----------------------------------------
+
+  If you require a VPROC entry for platform version identification, use the
+  following variables:
+
+### For Itanium
+
+    OPENSSL_VPROC_PREFIX=T0085H06
+     
+### For x86
+
+    OPENSSL_VPROC_PREFIX=T0085L01
+
+### Common Definition
+
+    export OPENSSL_VPROC=${OPENSSL_VPROC_PREFIX}_$(cat include/openssl/opensslv.h |\
+       sed -n -e 's/^ *# *define  *OPENSSL_VERSION_TEXT[^"]*"\([^"]*\)"/\1/p' |\
+       sed -e 's/[. ]/_/g' -e 's/[.-]/_/g' |\
+       grep -v fips \
+       )
+
+Example Configure Targets
+-------------------------
+
+  For OSS targets, the main DLL names will be `libssl.so` and `libcrypto.so`.
+  For GUARDIAN targets, DLL names will be `ssl` and `crypto`. The following
+  assumes that your PWD is set according to your installation standards.
+
+    ./Configure nonstop-nsx           --prefix=${PWD} --openssldir=${PWD}/ssl no-threads             --with-rand-seed=rdcpu ${CIPHENABLES} ${DBGFLAG} ${SYSTEMLIBS}
```

Comment:
Unless you explicitly intended it to be optimized for copy&paste (*), I'd prefer if you would break those extremely long lines an optimize for (terminal) readability. For example with a line continuation marker like this.

```
    ./Configure nonstop-nsx --prefix=${PWD} --openssldir=${PWD}/ssl no-threads  \
                --with-rand-seed=rdcpu ${CIPHENABLES} ${DBGFLAG} ${SYSTEMLIBS}
```

(*) _IMHO encouraging blind copy&paste prevents people to turn on their mind before pasting ;-)_